content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+```
+
+The currently logged user should have read and write access to the serial port over USB. This is done by adding the user to dialout group with the following command:
+
+```bash
+sudo usermod -a -G dialout $USER
+```
+
+#### Connecting to a Wi-Fi Access Point
+
+As an example, let's use the Wi-Fi configuration with the following command:
+
+```bash
+./tools/configure.sh esp32-devkitc:wifi
+```
+
+To change some configuration in the default configuration that was just loaded, use `menuconfig`:
+
+```bash
+make menuconfig
+```
+
+The `menuconfig` utility allows to enable peripherals, modify the NuttX kernel settings, enable support for device drivers, configure the *NuttShell* and the file system, enable example programs and set up the Wi-Fi credentials, for instance.
+
+To quickly connect to a known Wi-Fi network, navigate to `Application Configuration -> Network Utilities -> Network Initialization -> WAPI Configuration` and set the SSID (the name of the Wi-Fi Access Point) and the passphrase to connect to it.
+
+It is also necessary to set how to get a valid IP address after connecting to the Wi-Fi network. In this example, DHCP will be used to receive it from the network automatically. In `Application Configuration -> Network Utilities -> Network Initialization -> IP Address Configuration`, select `Use DHCP to get IP address`.
+
+Navigate to the `Exit` and, when prompted, save the current settings.
+
+Before building and flashing the firmware, it is necessary to build the bootloader to boot NuttX:
+
+```bash
+make bootloader
+```
+
+To build and flash the ESP32 board, use the following command (modifying the USB port accordingly):
+
+```bash
+make flash ESPTOOL_PORT=/dev/ttyUSB0 ESPTOOL_BINDIR=./
+```
+
+When flashing is complete, open a serial console and type `Enter`. For instance, using *picocom*:
+
+```bash
+picocom -b115200 /dev/ttyUSB0
+```
+
+Please check the following example that shows the device connecting to the Wi-Fi network and other commands that can be used in the *NuttShell*, like `ifconfig`:
+
+{{< asciinema key="demo" idleTimeLimit="2" speed="1.5" poster="npt:0:09" >}}
+
+It's worth noting that there is no need to pre-define the SSID/passphrase of the Wi-Fi network in `menuconfig`. NuttX provides the `wapi` application that manages the Wi-Fi subsystem. More information about `wapi` can be found in [NuttX documentation][3]. Experiment with `wapi` for scanning available access points, for instance:
+
+```bash
+wapi scan wlan0
+```
+
+### Conclusion
+
+NuttX is one of the most preferred RTOS for developers familiar with Linux interfaces (including the so-called "Embedded Linux") because of its POSIX-compliant interface. Most of the features covered by this introductory article rely on usual applications, like the *NuttShell* (or NSH: the shell system used in NuttX, similar to bash) and usual built-in commands, like `ping`. Stay tuned for more articles about NuttX (in special, for the question "Why using NuttX"?).
+
```

Comment:
Links are not visible at the end of the article. It's good to have them explicit, so that a reader can follow up after reading the article: https://developer.espressif.com/blog/simplify-embedded-projects-with-esp-bsp/#useful-links

Reply:
Added useful links (different pro citations along the text)